### ISSUE TITLE
[Feature Request] Search by Name in UObjectHook

### ISSUE THREAD

**Issue by armax9** [praydog/UEVR#69]
The existing functionality allows for searching parameters based on their class. Is it possible to implement the search by name?
Could be faster and more effective in a lot of cases.

**Comment by Warmotor** [comment]
Yes please, this. Finding the weapon mesh in R:RC took me hours!

**Comment by lunchanddinner** [comment]
I think this would be the #1 feature!
